src/simulation.jl

```diff
@@ -483,9 +514,28 @@ function link_contract_operators(d::SummationDecapode, con_dec_operators::Set{Sy
   contract_defs
 end
 
+# TODO: Allow user to overload these hooks with user-defined code_target
+function hook_LCO_generate_inplace_expr(computation_name, computation, ::cpu, float_type::DataType)
+  return :($(add_inplace_stub(computation_name)) = $(Expr(:call, :*, computation...)))
+end
+
+function generate_parentheses_multipy(list)
```

Comment:
Comment to explain what is happening here

Reply:
Done.